src/Inkluzitron/Services/GraphPaintingStrategy.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using Inkluzitron.Models;
+
+namespace Inkluzitron.Services
+{
+    public abstract class GraphPaintingStrategy : IDisposable
+    {
+        private bool _disposed;
+
+        public int ColumnCount { get; init; } = 5;
+        public Color BackgroundColor { get; set; } = Color.Black;
+        public int CategoryBoxPadding { get; set; } = 20;
+        public int CategoryBoxHeight { get; set; } = 350;
+
+        public Font GridLinePercentageFont { get; set; }
+        public Brush GridLinePercentageForegroundMajor { get; set; } = new SolidBrush(Color.FromArgb(0x70AAAAAA));
```

Comment:
Použití hexa :notlikethis:. Co takhle jako normální člověk použít něco jako `Color.FromArgb(0, 0, 0, 0)`...? :rtzW:

Reply:
Co se ti nelíbí na hexa? Jediný co na tom je lehce matoucí je to 0x70.